### ISSUE TITLE
Continue shrinking the Internal SGID

### ISSUE THREAD

**Issue by gregbunce** [agrc/planning-queue#337]
### Benefit

The benefit of continuing to shrink the Internal SGID is that we are not maintaining irrelevant data, and therefore this frees up our time to work on more important projects.

In the world of UGRC, this is an ongoing project that we should make progress on each quarter. That's the nature of of our business (keeping the data relevant).... datasets change, the stewards find better ways to serve them, etc.. In turn, we need to stay current and clean up the Internal so we're not serving outdated or irrelevant data.  

For this quarter I propose the data team take a look at datasets that have not had an update since 2020. We flag those datasets and then establish a path forward for them. Some may need to be deprecated, others may become a part of the SGID Index b/c the stewards are now serving their own web service (data).

### Acceptance Criteria

- [ ] Create a list of datasets in the Internal SGID that have not had an update since 2020
- [ ] Determine a path forward for those datasets (keep, deprecate, data-come-full-circle and steward has a web service, etc.)

### Risks

None

### Issue Reference

refs #

**Comment by gregbunce** [comment]
Rick and I are meeting May 21st to come up with a plan for this sprint 6 feature.

**Comment by gregbunce** [comment]
Rick and I decided to use some time in sprint 6 to create an outline on what the sgid looks like moving forward. We'll be working on that and then seeking feedback.

**Comment by rkelson** [comment]
We should create a list of feature layers that reside in AGOL only with no Internal layer. This idea is in response to removing State Fuel Sites from Internal while having it persist in AGOL and needing a strategy to backup AGOL only layers.

**Comment by rkelson** [comment]
ran this query on the SGID Index tab and it only came up with one layer, the Statewide parcel layer used in applications:
`storageType IN ('AGOL', 'cloud storage') And hostedBy = 'UGRC' And arcGisOnline = 'TRUE' And itemId IS NOT NULL `

refs [#325 ](https://github.com/agrc/porter/issues/325)

**Comment by steveoh** [comment]
Statewide parcels are a bit unique since we can recreate them from data in the internal database.

**Comment by gregbunce** [comment]
rick and i had some good discussions this quarter. as a data team, we've all also discussed this during our 'sgid: a plan for the future" meetings.

**Comment by gregbunce** [comment]
informed from the rating system project

**Comment by rkelson** [comment]
Other good candidates for removal from Internal and live only in AGOL:

indices.google_flight_blocks
indices.google_service_dates
indices.hexagon_service_dates
water.great_salt_lake_water_level_2016

geoscience.alluvual_fans - This needs to be updated first with https://drive.google.com/drive/folders/1Q-SYLHD7TOxu6k7NFd8lUJeHWll1Xf0K
- 'They conducted a new study of alluvial fans in 2023, and that is the latest alluvial fan data. She was saying that this 2023 version will be static going forward'

**Comment by stdavis** [comment]
These datasets that are removed from internal may be good candidates for adding the `backup` tag in AGOL so that they are backed up by [moonwalk](https://github.com/agrc/project-moonwalk).

**Comment by eneemann** [comment]
> We should create a list of feature layers that reside in AGOL only with no Internal layer. This idea is in response to removing State Fuel Sites from Internal while having it persist in AGOL and needing a strategy to backup AGOL only layers.

Is the DABS licenses layer one that only lives in AGOL?  I don't think it's in the SGID Index, but maybe it should be?  I think it would be a useful layer to publish and I think DABS puts out a spreadsheet of licenses and addresses routinely, so they probably wouldn't mind.  (Hopefully I'm not too far off topic)

**Comment by rkelson** [comment]
yeah it only lives in AGOL. I don't know if it is getting moonwalked. Not sure that I agree that it should be published. That would be a conversation to have with Janet and Jeremy.

**Comment by rkelson** [comment]
added `Backup` tag in AGOL to [DABS_GIS](https://utah.maps.arcgis.com/home/item.html?id=0909ac49fa404f1793862499e914caef)

**Comment by rkelson** [comment]
> These datasets that are removed from internal may be good candidates for adding the backup tag in AGOL so that they are backed up by [moonwalk](https://github.com/agrc/project-moonwalk).

I would think this is or should be part of the porter check boxes but good point

**Comment by eneemann** [comment]
> yeah it only lives in AGOL. I don't know if it is getting moonwalked. Not sure that I agree that it should be published. That would be a conversation to have with Janet and Jeremy.

Agreed, it's definitely a DABS decision.

**Comment by chriswnek** [comment]
Reviewed in backlog refinement.  Team would still like to work on but is not a high priority for next quarter.

**Comment by chriswnek** [comment]
Can revisit again for FY26 Q3.  In the meantime the team can chip away at individual checkbox items as time permits.
